Sage/carbon
# feat(confirm): make data tag props available on cancel and confirm buttons - FE-6251

This enhancement makes`data-role` and `data-element` tags available on the cancel and confirm buttons in this component

fixes #6374

### Proposed behaviour

Makes the data tag props available to be used via two new props called `cancelButtonDataProps` and `confirmButtonDataProps`.

### Current behaviour

Data tags cannot be used on either `cancel` or `confirm` buttons.

### Checklist




#### QA


### Additional context

N/A

### Testing instructions

- Should be able to pass custom `data-role` and `data-element` props to the `cancel` or `confirm` buttons. `Confirm with Custom Data Tags` example should demonstrate this.
- There should be no other regressions with functionality or styling with this component.
